app/(afterLogin)/(deal)/deals/activity/page.tsx

```
@@ -0,0 +1,320 @@
+
+const PersonMeta: React.FC<{ name: string; deal: string }> = ({
+  name,
+  deal,
+}) => (
+  <div className="text-xs text-slate-500 leading-tight">
+    <div>{name}</div>
+    <div className="text-slate-400">{deal}</div>
+  </div>
+);
+
+const Attachments: React.FC<{ items: string[] }> = ({ items }) => (
```

ESLint

[error] 'Attachments' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)